migtools/oadp-non-admin
# fix: Update status design with NAR

## Why the changes were made

Follow up of https://github.com/migtools/oadp-non-admin/pull/23 to:
- update workflow, that was changed by previous PRs
- add NonAdminRestore to design, that will be added by https://github.com/migtools/oadp-non-admin/pull/42

## How to test the changes made

[Read design](https://github.com/mateusoliveira43/oadp-non-admin/blob/fix/update-status-design/docs/design/nab_and_nar_status_update.md) and see if it is clear and the code follows it.
